manta-trusted-setup/src/groth16/test.rs

```diff
@@ -63,10 +63,10 @@ impl HasDistribution for Test {
 impl Pairing for Test {
     type Scalar = Fr;
     type G1 = G1Affine;
-    type G1Prepared = <Bls12_381 as PairingEngine>::G1Prepared;
+    type G1Prepared = <Bn254 as PairingEngine>::G1Prepared;
     type G2 = G2Affine;
-    type G2Prepared = <Bls12_381 as PairingEngine>::G2Prepared;
-    type Pairing = Bls12_381;
+    type G2Prepared = <Bn254 as PairingEngine>::G2Prepared;
+    type Pairing = Bn254;
```

Comment:
Like above use `Self::Pairing` as the argument in `<Self::Pairing as PairingEngine>::G2Prepared`